server/.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+on: [push, pull_request]
```

Comment:
Don't you need to CD into the server directory?

Reply:
maybe, its copy pasta :D